package_adafruit_index.json

```diff
@@ -5106,6 +5106,112 @@
               "version": "9.4.0"
             }
           ]
+        },
+        {
+          "name": "Adafruit SAMD Boards",
+          "architecture": "samd",
+          "version": "1.5.11",
+          "category": "Adafruit",
+          "url": "https://adafruit.github.io/arduino-board-index/boards/adafruit-samd-1.5.11.tar.bz2",
+          "archiveFileName": "adafruit-samd-1.5.11.tar.bz2",
+          "checksum": "SHA-256:3c27fa306e2e8c7d9067fd74d58cfec0b32daec57b5ef17a01660c1053b2c771",
+          "size": "4355649",
+          "help": {
+            "online": "https://forums.adafruit.com"
+          },
+          "boards": [
+            {
+              "name": "Adafruit Feather M0"
+            },
+            {
+              "name": "Adafruit Feather M0 Express"
+            },
+            {
+              "name": "Adafruit Metro M0 Express"
+            },
+            {
+              "name": "Adafruit Circuit Playground Express"
+            },
+            {
+              "name": "Adafruit Gemma M0"
+            },
+            {
+              "name": "Adafruit Trinket M0"
+            },
+            {
+              "name": "Adafruit ItsyBitsy M0"
+            },
+            {
+              "name": "Adafruit pIRkey M0"
+            },
+            {
+              "name": "Adafruit Metro M4"
+            },
+            {
+              "name": "Adafruit Grand Central M4"
+            },
+            {
+              "name": "Adafruit ItsyBitsy M4"
+            },
+            {
+              "name": "Adafruit Grand Central M4"
+            },
+            {
+              "name": "Adafruit Feather M4 Express"
+            },
+            {
+              "name": "Adafruit Hallowing M0"
+            },
+            {
+              "name": "Adafruit NeoTrellis M4"
+            },
+            {
+              "name": "Adafruit PyPortal M4"
+            },
+            {
+              "name": "Adafruit PyBadge M4"
+            },
+            {
+              "name": "Adafruit Metro M4 AirLift"
+            }
+          ],
+          "toolsDependencies": [
+            {
+              "packager": "arduino",
+              "name": "arm-none-eabi-gcc",
+              "version": "7-2017q4"
```

Comment:
did we want to bump this?

Reply:
yeah, it is part of the #52 , the SAMD official is upgraded to gcc v7. But it could definitely causes issue with other libraries. Just go through #47. I think we only need to update the bossac/openocd is enough. I will revert the gcc part of #52 and re-submit PR tomorrow.

Reply:
as @sta-c0000 mentionend, the platform.txt indeed specify the gcc v7 for compiling. This changes should install the correct gcc version, instead of relying on arduino:samd to install compiler for us

https://github.com/adafruit/ArduinoCore-samd/blob/master/platform.txt#L34